tests/integration/medusa_s3_integration_test.go

```diff
@@ -0,0 +1,60 @@
+package integration
+
+import (
+	. "github.com/k8ssandra/k8ssandra/tests/integration/steps"
+
+	. "github.com/onsi/ginkgo"
+)
+
+var (
+	medusaS3TestTable    = "medusa_test"
```

Comment:
Why are these separate from minioTestTable? Why not simply const( testTable = "medusa_test") in a single file?

Reply:
I'm a Go noob, so trying to make things work as my main goal. Happy to refactor as suggested, it's just that I'm not sure yet what's the variables scope, especially in the context of Ginkgo.

Reply:
If ginkgo modifies scopes, then it shouldn't be used.